model.go

```diff
@@ -234,14 +239,7 @@ func (v *Value) ShortSummary() string {
 	if v.Flag != nil {
```

Comment:
Instead of adding `placeholderFormatter` to the `Value`, can the formatter be passed through as a parameter here and elsewhere? (This may make everything a bit uglier though, so I'm not sure if it's a good idea).

Reply:
Yes that would be possible and I agree with you that it makes the code a little uglier.

Instead of `func (v *Value) FormatPlaceHolder() string ` we could go with a function:

```go
func FormatPlaceholder (ctx kong.Context, v *Value) string {

}```

And from kong.Context I would be able to get the configured formatter.